src/main/scala/io/iohk/ethereum/jsonrpc/server/http/JsonRpcHttpServer.scala

```diff
@@ -84,6 +84,16 @@ trait JsonRpcHttpServer extends Json4sSupport with RateLimit with Logger {
     }
   }
 
+  private def handleResponse(f: Task[JsonRpcResponse]): Task[(StatusCode, JsonRpcResponse)] = f map { jsonRpcResponse =>
+    jsonRpcResponse.error match {
+      case Some(JsonRpcError(error, _, _))
+          if List(JsonRpcError.InvalidRequest.code, JsonRpcError.ParseError.code, JsonRpcError.InvalidParams().code)
```

Comment:
For readability, this List of errors could be extracted to a constant at the class level or companion object

Reply:
Done